docs/api/cloudrootfs.md

```diff
@@ -4,7 +4,7 @@ cloud rootfs will package all the dependencies refers to the kubernetes cluster
 
 ```shell script
 .
-├── bin
+├── bin # binaries will be installed at all nodes' /usr/local/bin
```

Comment:
If we only store binaries in this directory, I am wondering  whether it is proper to container scripts here, like  `containerd-rootless-setuptool.sh`, `containerd-rootless.sh` and `kubelet-pre-start.sh`.

In addition, binary may not be compatible with plarforms(x86, arm). Then in the design, we make a ClusterImage to be compatible to all arch platforms or different ClusterImage compatible to different platforms. 

It could be better that we have a top-down through design, I think. 🐱

Reply:
Agree with that. It's weird to have a bunch of shell scripts in a directory called bin